Gemfile.lock

```diff
@@ -88,8 +79,14 @@ GEM
       jwt (>= 1.0, < 3.0)
       rails (>= 3.1)
       rest-client (>= 1.6.0)
+    aes_key_wrap (1.1.0)
     amazing_print (1.5.0)
+    apple_id (1.5.2)
```

Comment:
```
Fetching gem metadata from https://rubygems.org/.......
Resolving dependencies......
Bundler could not find compatible versions for gem "faraday":
  In snapshot (Gemfile.lock):
    faraday (= 1.10.3)

  In Gemfile:
    googleauth was resolved to 0.17.1, which depends on
      faraday (>= 0.17.3, < 2.0)

    apple_id (= 1.6.2) was resolved to 1.6.2, which depends on
      rack-oauth2 (~> 2.0) was resolved to 2.2.0, which depends on
        faraday (~> 2.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
```

Reply:
Interesting, there's a newer googleauth as well, but it's not installing because of google-api-client, which is version-locked at ~> 0.53

```
Fetching gem metadata from https://rubygems.org/.......
Resolving dependencies......
Bundler could not find compatible versions for gem "googleauth":
  In Gemfile:
    googleauth (= 1.5.2)

    google-api-client (~> 0.53) was resolved to 0.53.0, which depends on
      google-apis-core (~> 0.1) was resolved to 0.2.0, which depends on
        googleauth (~> 0.14)
```

Reply:
If you `bundle update apple_id googleauth google-api-client`, it will say
> Bundler attempted to update google-api-client but its version stayed the same
but will update apple_id anyway.

Reply:
Ok, can you commit it, then?

Reply:
And looks like google api client is used in `app/lib/update_global_activity_analytics.rb`. Looks like we're supposed to switch to the gem corresponding to the specific "service" we want: https://github.com/googleapis/google-api-ruby-client/commit/ded98b2afa1c59b73c0debe6565454120cad9546